File: TESTPizzeria/Pizzeria/src/main/java/com/sio/pizzeria/DTO/ProduitDTO.java
package com.sio.pizzeria.DTO;

public class ProduitDTO {

    private int idProduit;
    private String nomProduit;
    private double prix;
    private int idTypeProduit;

    public ProduitDTO(String nomProduit, double prix, int idTypeProduit) {
        this.nomProduit = nomProduit;
        this.prix = prix;
        this.idTypeProduit = idTypeProduit;
    }

    public int getIdProduit() {
        return idProduit;
    }

    public String getNomProduit() {
        return nomProduit;
    }

    public double getPrix() {
        return prix;
    }

    public int getIdTypeProduit() {
        return idTypeProduit;
    }

    public void setIdProduit(int idProduit) {
        this.idProduit = idProduit;
    }

    public void setNomProduit(String nomProduit) {
        this.nomProduit = nomProduit;
    }

    public void setPrix(double prix) {
        this.prix = prix;
    }

    public void setIdTypeProduit(int idTypeProduit) {
        this.idTypeProduit = idTypeProduit;
    }

}

File: TESTPizzeria/Pizzeria/src/main/java/com/sio/pizzeria/server/DTO/ProduitDTO.php
<?php

class ProduitDTO implements JsonSerializable {

    private $idProduit;
    private $nomProduit;
    private $prix;
    private $idTypeProduit;

    public function __construct($nomProduit, $prix, $idTypeProduit) {
        $this->nomProduit = $nomProduit;
        $this->prix = $prix;
        $this->idTypeProduit = $idTypeProduit;
    }

    public function getIdProduit() {
        return $this->idProduit;
    }

    public function getNomProduit() {
        return $this->nomProduit;
    }

    public function getPrix() {
        return $this->prix;
    }

    public function getIdTypeProduit() {
        return $this->idTypeProduit;
    }

    public function setIdProduit($idProduit): void {
        $this->idProduit = $idProduit;
    }

    public function setNomProduit($nomProduit): void {
        $this->nomProduit = $nomProduit;
    }

    public function setPrix($prix): void {
        $this->prix = $prix;
    }

    public function setIdTypeProduit($idTypeProduit): void {
        $this->idTypeProduit = $idTypeProduit;
    }

    public function jsonSerialize() {
        return array(
            'idProduit' => $this->idProduit,
            'nomProduit' => $this->nomProduit,
            'prix' => $this->prix,
            'idTypeProduit' => $this->idTypeProduit
        );
    }

}
